socorro/external/es/correlations.py

```diff
@@ -2,19 +2,41 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import datetime
 import json
 import os
+import hashlib
 
-from configman import Namespace, RequiredConfig
+from configman import Namespace
 from configman.converters import class_converter
 
+from socorro.analysis.correlations.correlations_rule_base import (
+    CorrelationsStorageBase,
+)
 
+
+# Needed for the ability to find the
+# mappings/correlations_index_settings.json file whose path is relative
+# to this file.
 HERE = os.path.dirname(os.path.abspath(__file__))
 
 
-class Correlations(RequiredConfig):
+def string_to_list(input_str):
```

Comment:
Already exists in configman: https://github.com/mozilla/configman/blob/master/configman/converters.py#L300

Use ``from configman import class_converter, str_to_list`` above.